docker-compose-shared.yml

```diff
@@ -0,0 +1,17 @@
+# Docker Compose over ride file for FreeCodeCamp
+#
+# Run with:
+#   docker-compose -f docker-compose.yml -f docker-compose-shared.yml up
+#
+version: "2"
```

Comment:
Are we better sticking to version 2? Are there issues with version 3?

I am out of the loop when it comes to docker

Reply:
Not sure actually, could be worth considering.

Reply:
Have bumped it to 3, as was done to staging in per https://github.com/freeCodeCamp/freeCodeCamp/pull/16825

Reply:
Bumped to 3, mostly because docker-compose.yml had been bumped. It seems version 3 is mostly related facilitating  Docker Swarm, which I would avoid.